code/datums/martial/berserker.dm

```diff
@@ -0,0 +1,130 @@
+/datum/martial_art/berserker
+	name = "Berserker"
+	var/datum/action/berserker/backbreak/backbreak = new/datum/action/berserker/backbreak()
```

Comment:
You don't need to specify the type with new if you already typecasted the variable.